pio/src/iSpindel.cpp

```diff
@@ -638,12 +648,105 @@ bool uploadData(uint8_t service)
     return sender.sendTCONTROL(my_server, my_port);
   }
 #endif // DATABASESYSTEM ==
-  return false;
+  
+#ifdef API_BLYNK
```

Comment:
There is a sender class to handle the upload, not the main file. Clear abstraction of concerns, not hackish

Reply:
I'll work on it and move to the sender class.